libcontainer/nsenter/cloned_binary.c

```diff
@@ -156,7 +156,8 @@ static int is_self_cloned(void)
 	ret = fcntl(fd, F_GET_SEALS);
 	if (ret >= 0) {
 		is_cloned = (ret == RUNC_MEMFD_SEALS);
-		goto out;
+		if (is_cloned)
+			goto out;
```

Comment:
This can actually be simplified to
```go
       if (ret == RUNC_MEMFD_SEALS) {
                is_cloned = 1;
                goto out;
        }
```

Reply:
Updated to ditch `ret` entirely.